test-utils/runtime-tester/README.md

```diff
@@ -0,0 +1,105 @@
+# Runtime test
+Runtime test is described by Scenario.
+Currently, scenario only supports one client, but you can specify the number of accounts through NetworcConfig.
+```rust
+#[derive(Serialize, Deserialize)]
+pub struct Scenario {
+    pub network_config: NetworkConfig,
+    pub blocks: Vec<BlockConfig>,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct NetworkConfig {
+    pub seeds: Vec<String>,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct BlockConfig {
+    pub height: BlockHeight,
+    pub transactions: Vec<TransactionConfig>,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct TransactionConfig {
+    pub nonce: Nonce,
+    pub signer_id: AccountId,
+    pub receiver_id: AccountId,
+    pub signer: InMemorySigner,
+    pub actions: Vec<Action>,
+}
+```
+
+Scenario can be loaded from a json file or constructed in rust code.
+```rust
+pub fn from_file(path: &Path) -> io::Result<Scenario>
+```
+
+Scenario::run tries to produce all the described blocks and if succeeded returns RuntimeStats.
+```rust
+pub fn run(&self) -> Result<RuntimeStats, Error>
+```
+
+RuntimeStats contain stats for every produced block. Currently, only block production time is supported.
+```rust
+#[derive(Serialize, Deserialize, Default, Debug)]
+pub struct RuntimeStats {
+    pub blocks_stats: Vec<BlockStats>,
+}
+
+#[derive(Serialize, Deserialize, Default, Debug)]
+pub struct BlockStats {
+    pub height: u64,
+    pub block_production_time: Duration,
+}
+```
+
+Be careful to remember, that block height should be positive and ascending.
+
+# Scenario Builder
```

Comment:
Extreme html pedantry: `#` generates a top-level heading `h1`. It is invalid to have more than one `h1`. The correct way is to use `# Moby Dick` for the title of the document, and `## Loomings` for the title of top-level chapter.